### DISCUSSION TITLE
In web component can we make route?

### DISCUSSION THREAD

**Discussion by jitendraP-ashutec** [lit/lit#3121] [Q&A] [answered]
Can we redirect to new page in using click event in web component? so If we using and frontend framework like angular then it should not refresh page. its only change route?  Like Tab bar with tab item which redirect to a new page.

**Comment by ynxtynxt** [comment]
Like using a router? https://github.com/lit/lit/tree/main/packages/labs/router

**Comment by e111077** [comment, marked as answer]
Lit doesn't do anything special for routing and changing links. If you're integrating in angular, I'd recommend firing an event from your lit component and your angular application handling that event and triggering its own router.

Otherwise you may want to look at what is offered natively by the browser like the [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) or the upcoming [Navigation API](https://developer.chrome.com/docs/web-platform/navigation-api/).

We will soon be adding a PWA starter kit to the repository which should have an example of simple routing, or as the other comment noted, take a look at the currently-experimental @lit-labs/router

**Reply by jitendraP-ashutec** [reply]
Ok Thanks @e111077
